vendor.conf

```diff
@@ -0,0 +1,50 @@
+# grpc and protobuf
```

Comment:
It's possible to use forks now. This file uses exactly same format as `hack/vendor.sh` in docker with all upsides and downsides :(